### PR TITLE
Helly space (S105) is contractible (P199)

### PR DESCRIPTION
**Pull request by GeoffreySangston**
Unrelated to this change, I did an ad hoc check and think the [Perfectly normal](https://topology.pi-base.org/spaces/S000105/properties/P000015), [Second countable](https://topology.pi-base.org/spaces/S000105/properties/P000027), and [Scattered](https://topology.pi-base.org/spaces/S000105/properties/P000051) traits can be automatically derived by pi-base.

### CODE REVIEW

**Comment by prabau** [comment]
Feel free to do other trait cleanup at the same time.

**Comment by GeoffreySangston** [comment]
Redundancy of perfectly normal trait: [π-Base, Search for `t2 + ~t5 + perfectly normal`](https://topology.pi-base.org/spaces?q=t2+%2B+%7Et5+%2B+perfectly+normal)

Redundancy of second countable trait: [π-Base, Search for `second countable + ~$T_5$ + t2 + compact `](https://topology.pi-base.org/spaces?q=second+countable+%2B+%7E%24T_5%24+%2B+t2+%2B+compact+)

Redundancy of scattered trait: [π-Base, Search for `scattered + t2 + ~t5 + compact + first countable`](https://topology.pi-base.org/spaces?q=scattered+%2B+t2+%2B+%7Et5+%2B+compact+%2B+first+countable)

Seems like a strict win to remove scattered and perfectly normal traits, since the references are just to the general reference chart in [Steen-Seebach]. The reference in [Steen-Seebach] for second countable does have an argument, so deleting it does have some cost. I'll delete it just to stick to some kind of policy.

**Comment by prabau** [comment]
I am not sure we have an official policy about that.  Sometimes some trait that can be derived has an interesting direct justification.  On the other hand, it's also good to see a minimal set of traits that imply the other ones.
Maybe @StevenClontz can comment.

**Comment by StevenClontz** [comment]
I think it's okay to remove an asserted trait that references a direct argument in *Counterexamples* (or any reference) when it can be deduced from other things known to pi-Base. Maybe supporting "see also" traits that aren't used in deduction but can annotate space/property relationships (e.g. a direct proof that a particular regular + second-countable space is metrizable) would be good down the road, but for now I think it's best to keep the footprint of our assertions small and use theorems to deduce as much as possible.

File: spaces/S000105/properties/P000199.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+space: S000105
+property: P000199
+value: true
+---
+
+Let $g$ be a function in Helly space. Since Helly space is closed under convex combination, $H(f, t) := (1-t)f + tg$ defines a homotopy $H : X \times [0, 1] \to X$ from the identity to a constant map.
```
Comment (StevenClontz) [inline spaces/S000105/properties/P000199.md:7]:
```suggestion
Let $\iota:X\to X$ be the identity and $z:X\to X$ be the constant map to the zero function $0\in X$. Then $H(f, t) := (1-t)f$ defines a continuous function $H : X \times [0, 1] \to X$ satisfying $H(f,0)=\iota(f)$ and $H(f,1)=z(f)$, showing the identity $\iota$ and constant map $z$ are [homotopic](https://en.wikipedia.org/wiki/Homotopy#Formal_definition).
```

Maybe this more directly fits the definition we have in P199?

**Review by StevenClontz — APPROVED** [review]
Result seems correct; consider my suggestion before merging.

File: spaces/S000105/properties/P000199.md (hunk 1 above)
Comment (GeoffreySangston) [reply spaces/S000105/properties/P000199.md:7]: Sounds good to me. Also I didn't notice it's a lot better to use $g = 0$.

**Pull request merged**